default.json

```diff
@@ -23,6 +26,14 @@
       "groupName": "Voiceflow dependencies",
       "matchPackagePrefixes": ["@voiceflow/"],
       "enabled": true
+    },
+    {
+      "groupName": "Voiceflow dependencies (automerge)",
+      "matchPackagePrefixes": ["@voiceflow/"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "matchCurrentVersion": "!/^0/",
+      "automerge": true,
```

Comment:
btw, you may want to look at the options included in the automerge presets: https://docs.renovatebot.com/presets-default/#automergeminor

they might be relevant to this

Reply:
Trying to figure out what the `pin` update type means 🤔

Unsure if we need to add the lockfile maintainence:
```
    {
      "groupName": "Lockfile maintenance",
      "matchUpdateTypes": ["lockFileMaintenance"],
      "automerge": true,
      "enabled": true
    },
```

Reply:
`pin` is when it pins a dependency range to an exact version (ex. ^1.0.0 to 1.0.0)

lockfile maintenance is unnecessary i believe. it's made redundant by pinning dependencies i think

Reply:
I'll leave as is for now then. We can add pin if we find it's needed, but I have a feeling it won't be for voiceflow deps since we pin them anyway.

Reply:
devs periodically forget to pin them. i would leave that one enabled

Reply:
It _seems_ we might be able to add `"pin", "pinDigest", "digest"` to the `matchUpdateTypes`.

Reply:
Maybe not, `digest` stuff might be for docker? Unsure.

Reply:
yeah digests sound like they're for docker images. let's just enable "pin" i think